core/src/wallet/cosmos/explorers/GaiaCosmosLikeBlockchainExplorer.cpp

```diff
@@ -274,5 +274,153 @@ namespace ledger {
                     .count());
         }
 
+        // Balances
+        /// Get Total Balance
+        FuturePtr<BigInt> GaiaCosmosLikeBlockchainExplorer::getTotalBalance(
+            const std::string &account) const
+        {
+            std::vector<FuturePtr<BigInt>> balance_promises({
+                getSpendableBalance(account),
+                getDelegatedBalance(account),
+                getUnbondingBalance(account),
+                getPendingRewards(account)
+                });
+
+            return async::sequence(getContext(), balance_promises)
+                .flatMap<std::shared_ptr<BigInt>>(getContext(), [](auto& vector_of_balances) {
+                    BigInt result;
+                    for (const auto balance : vector_of_balances) {
+                        result = result + *balance;
+                    }
+                    return FuturePtr<BigInt>::successful(std::make_shared<BigInt>(result));
+                });
+        }
+        /// Get total balance in delegation
+        FuturePtr<BigInt> GaiaCosmosLikeBlockchainExplorer::getDelegatedBalance(
+            const std::string &account) const
+        {
+            const auto endpoint = fmt::format("/staking/delegators/{}/delegations", account);
```

Comment:
Maybe define these URLs as constants in cosmos.hpp or somewhere ?

Reply:
Will do